Android/Java/app/build.gradle

```diff
@@ -43,7 +40,7 @@ dependencies {
     implementation 'androidx.multidex:multidex:2.0.1'
     implementation 'androidx.cardview:cardview:1.0.0'
     // Add IronSource Mediation SDK dependency.
-    implementation "com.ironsource.sdk:mediationsdk:$sdkVersion"
```

Comment:
did you check on a clean demo with no cache that the latest sdk is indeed downloaded and used?